brave/brave-core
# [Commander]: Translate all available commands

Resolves https://github.com/brave/brave-browser/issues/32704



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Enable brave://flags/#quick-commands
2. Open brave in a non-English locale
3. Press Ctrl+Space (to open the command palette)
4. The commands should be translated
